Google.Api.Generator.Tests/ProtoTests/Lro/Lro.proto

```diff
@@ -27,6 +27,15 @@ service Lro {
     };
     option (google.api.method_signature) = "name";
   }
+
+  // Test an LRO RPC with customized default polling settings.
+  rpc CustomDefaultPollingMethod(Request) returns(google.longrunning.Operation) {
```

Comment:
I know this is just a test method, but the combination of custom and default is always very confusing to me. Can you use default override instead?

Reply:
Or maybe PublishingSettingsPollingMethod? ServiceConfigPollingMethod? Happy to bike shed this :)